CONTRIBUTING.md

```diff
@@ -49,20 +53,23 @@ python3 -mvenv sandbox
 sandbox/bin/pip install -r requirements.txt
```

Comment:
I usually do this:

```
python3 -mvenv sandbox
source sandbox/bin/activate
pip install -r requirements.txt
```

Activation puts ``source/bin`` at the front of ``PATH``, making the commands shorter.

And, I actually use [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), which is a pain to setup (I do it once for each new employer), but puts virtualenvs in a folder outside of the repo. I don't recommend it for the general instructions for this project, `venv` works great and is built-in.

Reply:
Yeah, it seems everyone's got a different process/tools for virtualenvs.  I figured I could put the simplest possible here and everyone can translate to their process easily enough.  That's why I skipped `activate`, as the scope of its effect seems to confuse newcomers to Python (such as folks wanting to work on the frontend).